frontend/src/hooks/useCollaborateData.tsx

```diff
@@ -74,7 +89,12 @@ export const useCollaborateData = <Type,>(
     useEffect(() => {
         if (!cache || !data) return;
 
-        const equal = dequal(formatDequalData(cache), formatDequalData(data));
+        const cleanedCache = omitUnwantedData(cache, 'lastSeenAt');
```

Comment:
I'd test this entire hook. If you want a template for testing hooks you can check e.g. useChangeRequestsEnabled.test.tsx. I usually create a test component for the hook and check the test component.